lib/http.js

```diff
@@ -41,6 +41,8 @@ module.exports = function performFhServiceCall (opts, callback) {
       callback(null, null, body);
     }
   }
+  keycloak.getKeycloak().then(function(request) {
+    request(opts, onServiceCallComplete);
+  });
```

Comment:
Missing a catch here. Using the suggestion above, the catch should probably look something like:

```js
.catch(e => {
  log.error('the injectHeadersFn rejected with the following error: ')
  log.error(e)
})
```